hw/ip/otbn/doc/checklist.md

```diff
@@ -120,17 +120,17 @@ Testbench     | [CSR_CHECK_GEN_AUTOMATED][]           | Done        |
 Testbench     | [TB_GEN_AUTOMATED][]                  | N/A         |
 Tests         | [SIM_SMOKE_TEST_PASSING][]            | Done        |
 Tests         | [SIM_CSR_MEM_TEST_SUITE_PASSING][]    | Done        |
-Tests         | [FPV_MAIN_ASSERTIONS_PROVEN][]        | Not Started |
-Tool Setup    | [SIM_ALT_TOOL_SETUP][]                | Not Started |
-Regression    | [SIM_SMOKE_REGRESSION_SETUP][]        | In progress | This needs feedback from the Google side. [Draft PR](https://github.com/lowRISC/opentitan/pull/4189) available.
-Regression    | [SIM_NIGHTLY_REGRESSION_SETUP][]      | In progress | As above.
-Regression    | [FPV_REGRESSION_SETUP][]              | Not Started |
+Tests         | [FPV_MAIN_ASSERTIONS_PROVEN][]        | N/A         |
+Tool Setup    | [SIM_ALT_TOOL_SETUP][]                | Done        |
+Regression    | [SIM_SMOKE_REGRESSION_SETUP][]        | Done        |
+Regression    | [SIM_NIGHTLY_REGRESSION_SETUP][]      | Done        |
+Regression    | [FPV_REGRESSION_SETUP][]              | N/A         |
 Coverage      | [SIM_COVERAGE_MODEL_ADDED][]          | Done        | No sub-blocks, so auto-generated framework will work
```

Comment:
Looks like this comment pertains to the next item?

Reply:
Hmm, git blame shows that I wrote this, but I have no memory of doing so(!). I think the point was that the SIM_COVERAGE_MODEL_ADDED has a note about black-boxing prim submodules. But (given that you didn't understand what I meant, and I'm not absolutely sure either) I'm just going to get rid of it.